test/functional/api/cas/casadm.py

```diff
@@ -288,7 +288,7 @@ def set_param_cleaning_acp(cache_id: int, wake_up: int = None, flush_max_buffers
     output = TestRun.executor.run(
         set_param_cleaning_acp_cmd(
             cache_id=str(cache_id),
-            wake_up=str(wake_up) if wake_up else None,
+            wake_up=str(wake_up) if wake_up is not None else None,
```

Comment:
Is it necessary? You didn't change similar line below and I think it works.

Reply:
'wake_up_time' in ACP ranges from 0 not 1 like in ALRU, so when it is set to 0 this 'if' statement treats it like False and don't pass that 0 value to 'set_param_cleaning_acp_cmd' function. That change I've made strictly compare wake_up to None, so when it's 0 then everything works as it should.

Reply:
Thanks for explanation.